.stylelintrc.json

```diff
@@ -3,13 +3,14 @@
   "plugins": ["stylelint-use-logical"],
   "rules": {
     "custom-property-pattern": null,
+    "no-empty-source": null,
     "declaration-no-important": true,
-    "max-nesting-depth": [5, 
+    "max-nesting-depth": [5,
```

Comment:
Looks like I didn't update this to 3. 
I had it at five to get the build working. All the files should be updated now. 

Can you change to "3"?

Reply:
Done